app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/core/Declaration.scala

```diff
@@ -189,6 +190,7 @@ case class DeclarationJourney(sessionId: SessionId,
   val declarationIfRequiredAndComplete: Option[Declaration] = {
     val discardedAnswersAreCompleteAndRequireADeclaration =
       maybeGoodsDestination.isDefined &&
+        maybeGoodsRouteDestination.contains(No) &&
```

Comment:
a clearer name would make this easier to understand...

Reply:
However maybeGoodsRouteDestination is an optional question. It only has to be answered if  maybeGoodsDestination.contains(NorthernIreland).

Think this was a problem with the acceptance criteria.

I suspect this change makes it impossible to complete a declaration which doesn't go via Northern Ireland?

Reply:
Maybe we should re-work the Acceptance Criteria...